mu-plugins/blocks/global-header-footer/header.php

```diff
@@ -89,51 +97,8 @@
 	<!-- /wp:navigation -->
 
 	<?php if ( $show_search ) : ?>
-	<!--
-		The search block is inside a navigation submenu, because that provides the exact functionality the design
-		calls for. It also provides a consistent experience with the primary navigation menu, with respect to
-		keyboard navigation, ARIA states, etc. It also saves having to write custom code for all the interactions.
-	-->
```

Comment:
Was removing the above comment intentional? It's not related to how the search block was forked, and still seems like helpful documentation to me.

Reply:
I thought it was outdated since the submenu wrapper was removed in #119, but I guess it could be referring to the nav wrapper itself? Feel free to put it back, if so.

Reply:
Yeah, I think it's there to address the confusion over why a search box would be inside a menu.

Reply:
restored in 1951c1643d1bcfa999fdd927c6fb6e299c12287e